### PR TITLE
BIP21 Support for Wallet spending

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

File: BTCPayServer/Controllers/WalletsController.cs [hunk 1]
```diff
@@ -462,6 +462,50 @@ private async Task<bool> CanUseHotWallet()
             if (network == null || network.ReadonlyWallet)
                 return NotFound();
             vm.SupportRBF = network.SupportRBF;
+            
+            if (!string.IsNullOrEmpty(bip21))
```
Comment (rockstardev) [inline BTCPayServer/Controllers/WalletsController.cs]: I would recommend this extracted into separate `SendBip21` method. https://martinfowler.com/bliki/FlagArgument.html

Comment (Kukks) [reply BTCPayServer/Controllers/WalletsController.cs]: done

**Comment by NicolasDorier** [comment]
seems good. Can you add a test in CanManageWallet?

**Comment by Kukks** [comment]
@NicolasDorier  Added test

**Pull request merged**